_posts/2021-10-03-get-started-with-APIs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Get Started with APIs"
+description: "After reading  this blog post you will get to know what, why and how about APIs and for sure you will feel confident about this topic "
+category: Application Development
```

Comment:
category would be backend instead of application development

Reply:
`backend` to be specific